tests/stress/stress_test.go

```diff
@@ -0,0 +1,245 @@
+// Copyright 2020 ChainSafe Systems (ON) Corp.
+// This file is part of gossamer.
+//
+// The gossamer library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The gossamer library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the gossamer library. If not, see <http://www.gnu.org/licenses/>.
+
+package rpc
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/ChainSafe/gossamer/dot/rpc/modules"
+	"github.com/ChainSafe/gossamer/tests/rpc"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	NETWORK_SIZE_STR               = os.Getenv("NETWORK_SIZE")
+	GOSSAMER_INTEGRATION_TEST_MODE = os.Getenv("GOSSAMER_INTEGRATION_TEST_MODE")
+	pidList                        = make([]*exec.Cmd, 3)
+	keyList                        = []string{"alice", "bob", "charlie", "dave", "eve", "fred", "george", "heather"}
+	dialTimeout                    = 60 * time.Second
+	httpClientTimeout              = 120 * time.Second
+	rpcHost                        = "0.0.0.0"
+
+	transport = &http.Transport{
+		Dial: (&net.Dialer{
+			Timeout: dialTimeout,
+		}).Dial,
+	}
+	httpClient = &http.Client{
+		Transport: transport,
+		Timeout:   httpClientTimeout,
+	}
+)
+
+// runGossamer will start a gossamer instance and check if its online and returns CMD, otherwise return err
+func runGossamer(t *testing.T, nodeNumb int, dataDir string) (*exec.Cmd, error) {
+
+	currentDir, err := os.Getwd()
+	if err != nil {
+		return nil, err
+	}
+
+	gossamerCMD := filepath.Join(currentDir, "../..", "bin/gossamer")
+
+	//TODO: enable genesis file to be configured via args
+	//TODO: enable [core] authority and roles via args
```

Comment:
@ryanchristo @noot @edwardmack any ideas how to get this sorted via cmd line ?

Reply:
you mean with flags?

Reply:
yes

Reply:
there is a --genesis flag for use with `init` and there is a --roles flag for use with the base command, is that what you are looking for?

Reply:
ok I fixed it now, please re-check